### ISSUE TITLE
Pursuit external link CSS unsupported character

### ISSUE THREAD

**Issue by emiel** [purescript/purescript#3840]
The Pursuit CSS code in the PureScript compiler includes a rule that is intended to prefix a link with an arrow (I'm presuming to indicate an external link). The character https://unicode-table.com/en/1F852/ doesn't appear to be supported by most fonts and displays as an unsupported character in Firefox (See attached screenshot).

https://github.com/purescript/purescript/blob/0aaa49070829f5a0a6bbdeb8541988a05dc93fd8/app/static/pursuit.css#L209

Here is example of where the issue can be seen (Firefox displays an unsupported char whereas other browsers simply do no render the character):
https://pursuit.purescript.org/packages/purescript-affjax/10.0.0

Some options that come to mind:

- Is the arrow needed at all?
- Use a character that is supported in most fonts
- Use a custom font
- Others...

On a side note preparing a fix for this seems a bit convoluted. The CSS code for Pursuit is included in the PureScript compiler. One would need to patch the compiler, release and then rollout to Pursuit. Would be great to keep Pursuit CSS code with Pursuit itself. Though I know very little about how all this is setup... ;) Related issue: https://github.com/purescript/purescript/issues/3016

![purescript affjax   Pursuit](https://user-images.githubusercontent.com/37766/78979499-49761e80-7b1c-11ea-81dc-7dcee5d265b2.png)

**Comment by hdgarrood** [comment]
Thanks for the report! The CSS is inside the compiler because it is needed for use in `purs docs`, so that you can generate and view generated HTML documentation without needing to upload to Pursuit first. Replacing the arrow with a more common one or just removing it both seem acceptable to me. I think I’d prefer not to bother with a custom font.

**Comment by emiel** [comment]
That makes sense, thanks for clarifying.

I think I was wrong in my assumption that the arrow was meant to indicate external links. Only "code" links (using back-ticks) are prefixed with a "right arrow", whether the link is internal or external. So I'm not sure what the original intent was.

```
[`code`](url)
```

Another place where the issue is very apparent is on [user help](https://pursuit.purescript.org/help/users) page.

**Comment by hdgarrood** [comment]
I think the idea is that the arrow identifies the thing as a link, because it's not otherwise very clear that `<code>` blocks are links unless you hover over them.

**Comment by hdgarrood** [comment]
Changing the rule to
```css
a > code::before {
  content: "→";
  margin: 0 0.5em 0 0.2em;
  position: relative;
  bottom: 0.1em;
}
```
seems to work okay.

**Comment by hdgarrood** [comment]
@grossbart do you have any thoughts on this?

**Comment by grossbart** [comment]
I agree that we should not depend on an additional font just for this, so switching to a more supported character makes total sense to me. 👍

**Comment by emiel** [comment]
@hdgarrood The arrow doesn't look great on Firefox (macos). :/

![Screenshot_2020-04-10 Pursuit](https://user-images.githubusercontent.com/37766/79005396-10a76b00-7b57-11ea-8e1d-78954f0092ab.png)

Perhaps it is worth considering distinguishing "code" and "code links" another way. Maybe a different color? Say normal "code" is black and "code links" are blueish. Makes sense to make all links on the site the same color to convey what is an actual link (without underlining). I'm afraid the arrows might actually be a bit distracting anyway.

Some examples:

* https://docs.python.org/3/library/stdtypes.html?highlight=frozenset#memoryview.format
* https://doc.rust-lang.org/std/collections/index.html#iterators

Should an arrow be desired there is always SVG. Just thinking out loud here.

**Comment by grossbart** [comment]
Thanks for those links, @emiel, good ideas there. If we'd apply a Python-like styling, it would look like the screenshot; makes sense to me. In addition to what the Rust docs to, Python removes the background; either works, but I think the Python variant would be more clear in our context, because the navigation links are also coloured.

<img width="650" alt="grafik" src="https://user-images.githubusercontent.com/10811/79014500-53733e00-7b6b-11ea-8bc7-d071f7824e7e.png">

This is the CSS diff – if you could apply this, @hdgarrood, I would be thankful, as I don't have it running on my machine at the moment …

```diff
- a > code::before {
a > code::before, a > code::after {
-  content: "🡒";
-  content: "";
-  content: " attr";
-  letter-spacing: 0.33em;
+  content: "\00a0";
+  letter-spacing: -0.6em;
}

a > code {
-  font-weight: normal;
+  font-weight: bold;
+  background: transparent;
+  color: #c4953a;
}

a:hover > code {
-  color: #c4953a;
+  color: #7b5904;
}
```
